packages/viewer/src/scenes/camera.ts

```diff
@@ -67,9 +67,9 @@ export class Camera implements FrameCamera.FrameCamera {
    * Queues the rendering for a new frame using this camera. The returned
    * promise will resolve when a frame is received that contains this camera.
    */
-  public async render(): Promise<void> {
+  public async render(animation?: Animation.Animation): Promise<void> {
```

Comment:
I'd maybe rather have an object like this instead:

```ts
interface CameraRenderOptions {
  animation: Animation.Animation;
}
```

This'd more easily allow us to add additional fields without possible breaking changes.